client/src/components/users/InvitationPopoverItem.vue

```diff
@@ -89,6 +123,68 @@ async function copyLink(invitation: UserInvitation): Promise<void> {
     );
   }
 }
+
+async function sendEmail(invitation: UserInvitation): Promise<void> {
+  sendEmailDisabled.value = true;
+  const answer = await askQuestion('UsersPage.invitation.sendEmailTitle', 'UsersPage.invitation.sendEmailMessage', {
+    yesText: 'UsersPage.invitation.sendEmail',
+  });
+
+  if (answer === Answer.No) {
+    sendEmailDisabled.value = false;
+    return;
+  }
+  const result = await inviteUser(invitation.claimerEmail);
+  if (result.ok && result.value.emailSentStatus === InvitationEmailSentStatus.Success) {
+    props.informationManager.present(
+      new Information({
+        message: {
+          key: 'UsersPage.invitation.inviteSuccessMailSent',
+          data: {
+            email: invitation.claimerEmail,
+          },
+        },
+        level: InformationLevel.Success,
+      }),
+      PresentationMode.Toast,
+    );
+    setTimeout(() => {
+      sendEmailDisabled.value = false;
+    }, 5000);
+  } else {
+    let message: Translatable = '';
+    if (result.ok) {
+      message = 'UsersPage.invitation.inviteSuccessNoMail';
```

Comment:
This error message is out of place here: the only goal for this button is to send an email, this error message indicates that the invitation has been created but that the email couldn't have been sent. Might change it to something only indicating that the email couldn't be sent

Reply:
Yeah I got lazy, didn't want to add another translation and I was hoping nobody would catch that.